client/remoterepo.c

```diff
@@ -122,6 +123,14 @@ TDNFDownloadFile(
     char *pszUserPass = NULL;
     /* lStatus reads CURLINFO_RESPONSE_CODE. Must be long */
     long lStatus = 0;
+    metalink_error_t r;
```

Comment:
please use meaningful variable names. metalink_error for eg.

Reply:
Done, Thanks!